glide.yaml

```diff
@@ -56,7 +56,7 @@ import:
 - package: golang.org/x/net
   version: ab5485076ff3407ad2d02db054635913f017b0ed
 - package: github.com/gorilla/mux
-  version: 18fca31550181693b3a834a15b74b564b3605876
+  version: 1.6.1
```

Comment:
We were on an old version which was missing a number of testing features so I brought up us to the latest tag.

Reply:
Do you want to use `^1.6.1` instead of pinning it to a single version?

Reply:
Ping

Reply:
We discussed this offline and I said I would rather keep it pinned to a version.